packages/popper/src/useModifier.ts

```diff
@@ -0,0 +1,44 @@
+import { isArray } from '@vue/shared'
+
+interface IBuildModifierOptions {
+  arrowOffset: number
+  arrowRef: HTMLElement
+  boundariesPadding: number
+  cutoff: boolean
+  offset: [number, number] | number
+  showArrow: boolean
+  fallbackOptions: Record<string, unknown>
+}
+
+export default (options: IBuildModifierOptions) => {
+  const offset = isArray(options.offset) ? options.offset.slice(0, 2) : [0, options.offset]
+  const modifiers = [
```

Comment:
can we make this configs outside the function or it will be initialized every time when useModifier is called